packages/core-mobile/app/utils/publicKeys.ts

```diff
@@ -0,0 +1,36 @@
+import { NetworkVMType } from '@avalabs/vm-module-types'
+
+export const emptyAddresses = (): Record<NetworkVMType, string> => ({
+  [NetworkVMType.AVM]: '',
+  [NetworkVMType.BITCOIN]: '',
+  [NetworkVMType.CoreEth]: '',
+  [NetworkVMType.EVM]: '',
+  [NetworkVMType.HVM]: '',
+  [NetworkVMType.PVM]: '',
+  [NetworkVMType.SVM]: ''
+})
+
+export const findPublicKey =
+  (path: string, curve: Curve) => (pk: AddressPublicKey) =>
+    pk.derivationPath === path && pk.curve === curve
+
+export enum Curve {
+  Secp256k1 = 'secp256k1',
+  Ed25519 = 'ed25519'
+}
```

Comment:
```suggestion
export enum Curve {
  SECP256K1 = 'secp256k1',
  ED25519 = 'ed25519'
}
```
let's follow this format for all enums

Reply:
fixed